package.json

```diff
@@ -51,6 +57,7 @@
     "jest": "^22.4.2"
   },
   "dependencies": {
+    "app-root-dir": "^1.0.2",
```

Comment:
is `app-root-dir` required? Can't we avoid this as we always now the path? (Can we get the pat from `process.cwd()`?)
I would prefer to not include another dependency for a simple task as getting the current directory: if it is an electron app we have a predictable behavior.

Reply:
Doesn't work without it...what we can do though is copy this script in our `source` dir : https://github.com/philidem/node-app-root-dir/blob/master/lib/index.js
Should I do that?

Reply:
Lets keep the dependency, and merge this.
We will try to improve this later.